### ISSUE TITLE
ERRORS found in /var/log/bigbluebutton

### ISSUE THREAD

**Issue by ali-alhaidary** [bigbluebutton/bigbluebutton#15160]
**Describe the bug**
ERRORS found in /var/log/bigbluebutton

**To Reproduce**
use bbb-conf --watch
See error
ss                      80      443
*               1       0
grep: Invalid regular expression
grep: Invalid regular expression
[               0       0
27.0.0.1                1       0
7.0.0.1]                4       0
  -- ERRORS found in /var/log/bigbluebutton/* --
/var/log/bigbluebutton/archive-07960a83e08e8ad2769b7bab87b49ca01673818b-1653083668172.log:E, [2022-05-20T22:06:12.417521 #4490] ERROR -- : Failed to download fi
le: Failed to open TCP connection to 127.0.0.1:9002 (Connection refused - connect(2) for "127.0.0.1" port 9002)
/var/log/bigbluebutton/archive-07960a83e08e8ad2769b7bab87b49ca01673818b-1653084114766.log:E, [2022-05-20T22:06:13.727655 #4681] ERROR -- : Failed to download fi
le: Failed to open TCP connection to 127.0.0.1:9002 (Connection refused - connect(2) for "127.0.0.1" port 9002)
/var/log/bigbluebutton/archive-07960a83e08e8ad2769b7bab87b49ca01673818b-1653084560487.log:E, [2022-05-20T22:12:42.197005 #6015] ERROR -- : Failed to download fi
le: Net::ReadTimeout
/var/log/bigbluebutton/archive-07960a83e08e8ad2769b7bab87b49ca01673818b-1653088962147.log:E, [2022-05-20T23:34:10.730078 #8175] ERROR -- : Failed to download fi

BigBlueButton Server 2.5.0 (3018)

**Comment by easy-bios** [comment]
Hi, I had some refused connections, but it is gone in 2.5.1.
Can you confirm?

```
2022/06/10 06:06:26 [error] 473716#473716: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 168.119.xxx.xx, server: [bbb-250.xxxxxxxxxxx.com](http://bbb-250.xxxxxxxxxxx.com/), request: "GET /bigbluebutton/api HTTP/2.0", upstream: "http://127.0.0.1:8090/bigbluebutton/api", host: "bbb-250. [xxxxxxxxxxx.com](http://xxxxxxxxxxx.com/)"
```

**Comment by ali-alhaidary** [comment]
My question here is: What file it is trying to download?

**Comment by ali-alhaidary** [comment]
@easy-bios No, the same error still reported in v2.5.1, do you know what file it is trying to download?

**Comment by easy-bios** [comment]
Can you please check if the shared Notes are working on your server?
https://docs.bigbluebutton.org/dev/recording.html#captured-files

**Comment by ali-alhaidary** [comment]
> So i guess this is the issue. You have Notes disabled but something in your meeting or recording is trying to fetch something from Port 9002 (NOTES service). Maybe at this point some Dev should take a look on it with disabled Notes.

@antobinary  can you help please?
